modules/ingest-common/src/yamlRestTest/resources/rest-api-spec/test/ingest/121_grok_capture_all_matches.yml

```diff
@@ -39,6 +39,9 @@ teardown:
 
 ---
 "Test Grok Pipeline with capture_all_matches enabled":
+  - skip:
+      version: " - 3.2.99"
```

Comment:
I'm not as familiar with the syntax in the yaml rest tests, but do you know why it's `- 3.2.99` instead of `<3.3.0`? Approved the PR as I see the same syntax in other tests.

Reply:
Thanks for your comment, from the definition, the right bound is included, see https://github.com/opensearch-project/OpenSearch/tree/main/rest-api-spec/src/main/resources/rest-api-spec/test#skipping-tests.

Reply:
Got it, I think we should change that in the future but definitely out of scope of this PR. The existing setup means we have to make up versions that don't exist which doesn't make sense.